### DISCUSSION TITLE
Training with multiple GPU on Kaggle

### DISCUSSION THREAD

**Discussion by lcoandrade** [azavea/raster-vision#1814] [Q&A]
I'm running this Kaggle [notebook](https://www.kaggle.com/code/luizclaudioandrade/rastervision-101) with GPU T4x2, but, while training, I just see 1 GPU working.

What can be done to use both GPU's?
<img width="1342" alt="Captura de Tela 2023-06-28 às 07 50 21" src="https://github.com/azavea/raster-vision/assets/8769816/f035f02c-d806-4431-999d-7d7e3301a92e">

**Comment by AdeelH** [comment]
Multi-GPU training is not currently supported (#1352). You can get around that by [using Lightning](https://docs.rastervision.io/en/0.20/usage/tutorials/lightning_workflow.html) instead of RV `Learner`s.
